vllm/v1/worker/gpu_model_runner.py

```diff
@@ -763,6 +773,7 @@ def _update_states(self, scheduler_output: "SchedulerOutput") -> None:
             if not is_last_rank:
                 # Add new_token_ids to token_ids_cpu.
                 start_token_index = num_computed_tokens
+                assert new_token_ids is not None
```

Comment:
Also, this is weird

Reply:
This is needed for the type checker after changing `req_data.new_token_ids` to `list[list[int] | None]`.